### PR TITLE
Changes the elevator music to system shock

### PR DESCRIPTION
**Pull request by kevinz000**
Agh ouch, someone please propose an alternative.

### CODE REVIEW

**Comment by Keekenox** [comment]
I like the music, boooooo.

**Comment by yourdoom9898** [comment]
Didn't we have a PR that changed it ages ago to the system shock music?

**Comment by avabee** [comment]
It got reverted somehow.

File: maps/tether/tether_areas.dm [hunk 1]
```diff
@@ -24,7 +24,6 @@
 // Elevator areas.
 /area/turbolift
 	delay_time = 2 SECONDS
```
Comment (avabee) [inline maps/tether/tether_areas.dm]:
```suggestion
	delay_time = 2 SECONDS
	forced_ambience = list('modular_citadel/sound/music/elevator.ogg')
```

**Pull request closed without merging**